tests/llmobs/conftest.py

```diff
@@ -212,3 +215,53 @@ def mock_ragas_evaluator(mock_llmobs_eval_metric_writer, ragas):
     LLMObsMockRagas.return_value = 1.0
     yield RagasFaithfulnessEvaluator
     patcher.stop()
+
+
+@pytest.fixture
+def tracer():
+    return DummyTracer()
+
+
+@pytest.fixture
+def llmobs_env():
+    return {
+        "DD_API_KEY": "<default-not-a-real-key>",
+        "DD_LLMOBS_ML_APP": "unnamed-ml-app",
+    }
+
+
+class TestLLMObsSpanWriter(LLMObsSpanWriter):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.events = []
+
+    def enqueue(self, event):
+        self.events.append(event)
+
+
+@pytest.fixture
+def llmobs_span_writer():
+    yield TestLLMObsSpanWriter(interval=1.0, timeout=1.0)
```

Comment:
Does this get re-instantiated per test? If not, do we need to reset the writer events list after each test?

Reply:
yes, reinstantiated per test so no state being carried between tests